single-line-diagram-core/src/test/java/com/powsybl/sld/AbstractTestCase.java

```diff
@@ -91,7 +91,7 @@ protected void writeDebugFilesFromJson(String refMetdataName, StringWriter write
     }
 
     protected void writeToFileInDebugDir(String filename, StringWriter content) {
-        File debugFolder = new File(System.getProperty("user.home"), ".powsybl");
```

Comment:
Is the '/' character working for all platforms?
This makes me realize `java.io.File` is a bit oldish, I think while we're at it we should switch to `java.nio.file`. Then we can use Path.of(x, y, z). And then use `Files.createDirectories` and `Files.newBufferedWriter`.